### PR TITLE
removing and adding data into to do list

### PR DESCRIPTION
**Pull request by nimplay**
summary:

- [ ] add system 
- [ ] removing system
- [ ] save system

### CODE REVIEW

File: src/modules/secondbar.js [hunk 1]
```diff
@@ -13,6 +18,26 @@ const secondBar = () => {
...
   form.appendChild(inputText);
   form.appendChild(addBtn);
+  let lastIndex;
+  let lastDato;
+  addBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (inputText.value) {
+      if (toDoTasks.length === 0) {
+        lastIndex = 0;
+      } else {
```
Comment (codepantha) [inline src/modules/secondbar.js:28]: - [ ] An error page shows when if a user attempts to add a task by pressing the "Enter" key on the keyboard as shown below 👇 
image
In order to fix this issue, you can add a `keydown` event listener and check if the `code === 'Enter'`

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @nimplay 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Linter checks
✔️ Correct GitHub flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] The code in `removeTask.js`, `deleteComplete.js`, `get-task.js`, and `edit.js` could be combined into a single js file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nimplay** [comment]
Sumary:

- [ ] Window Alert was changed
- [ ] page error presing enter was fixed
- [ ] improve apparience

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @nimplay, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**